docs/products/cassandra/get-started.md

```diff
@@ -4,34 +4,418 @@ sidebar_label: Get started
 
-## Create a service with the Aiven CLI
+## Create a service
 
-To create a service using the
-[Aiven CLI](https://github.com/aiven/aiven-client), decide on the service plan, cloud
-provider, and region for your new service and run the `avn service create` command.
+<Tabs groupId="group1">
+<TabItem value="1" label="Console" default>
+[Create an Aiven for Apache Cassandra® service](/docs/platform/howto/create_new_service)
+in the [Aiven Console](https://console.aiven.io).
+</TabItem>
+<TabItem value="2" label="Terraform">
+
+1. [Create an authentication token](/docs/platform/howto/create_authentication_token).
+
+1. Create the `cassandra-sample.tf` file for the `aiven` provider configuration and
+   the `aiven_cassandra` resource.
+
+   ```hcl
+   variable "aiven_token" {
+     type = string
+   }
+
+   variable "aiven_project_name" {
+     type = string
+   }
+
+   terraform {
+     required_providers {
+       aiven = {
+         source  = "aiven/aiven"
+         version = ">=4.0.0, <5.0.0"
+       }
+     }
+   }
+
+   provider "aiven" {
+     api_token = var.aiven_token
+   }
+
+   resource "aiven_cassandra" "example_cassandra" {
+     project                 = var.aiven_project_name
+     cloud_name              = "google-europe-west1"
+     plan                    = "startup-4"
+     service_name            = "cassandra-sample"
+     maintenance_window_dow  = "friday"
+     maintenance_window_time = "23:00:00"
+     termination_protection  = false
+
+     cassandra_user_config {
+       migrate_sstableloader = true
+       service_log           = false
+     }
+   }
+
+   output "cassandra_service_host" {
+     value = aiven_cassandra.example_cassandra.service_host
+   }
+
+   output "cassandra_service_port" {
+     value = aiven_cassandra.example_cassandra.service_port
+   }
+
+   output "cassandra_service_username" {
+     value = aiven_cassandra.example_cassandra.service_username
+   }
+
+   output "cassandra_service_password" {
+     value     = aiven_cassandra.example_cassandra.service_password
+     sensitive = true
+   }
+    ```
+
+1. Create the `terraform.tfvars` file for assigning actual values to your previously
+   declared variables.
+
+   ```hcl
+   aiven_token    = "AIVEN_TOKEN"
+   aiven_project_name = "PROJECT_NAME"
+   ```
+
+1. Run `terraform init` > `terraform plan` > `terraform apply`.
+
+1. Store Terraform outputs in environment variables so that they can be used for
+   [connecting](#connect-to-service):
+
+   ```bash
+   CASSANDRA_HOST="$(terraform output -raw cassandra_service_host)"
+   CASSANDRA_PORT="$(terraform output -raw cassandra_service_port)"
+   CASSANDRA_USER="$(terraform output -raw cassandra_service_username)"
+   CASSANDRA_PASSWORD="$(terraform output -raw cassandra_service_password)"
+   ```
+
+</TabItem>
+<TabItem value="3" label="K8s">
+
+Create an Aiven for Apache Cassandra service using the Aiven Operator for Kubernetes.
```

Comment:
We do not have a similar intro sentence for TF, let's do similiar things? Either delete this line or add one to the TF tab?